app/server.py

```diff
@@ -54,42 +54,8 @@
     allow_headers=["*"],
 )
 
-endpoints = SmartAPI(OPENAPI_SERVER_MATURITY).get_operations_endpoints()
+# Global services dict. Is set on initialization
```

Comment:
Let's add a little more detail to this comment.

It is set in refresh_services() through a global reference and called through App.on_event("startup")